velox/exec/TableWriter.cpp

```diff
@@ -120,13 +120,15 @@ void TableWriter::addInput(RowVectorPtr input) {
     return;
   }
 
+  traceInput(input);
```

Comment:
Not sure if we could invoke this in 
```
              CALL_OPERATOR(
                  nextOp->addInput(intermediateResult),
                  nextOp,
                  curOperatorId_ + 1,
                  kOpMethodAddInput);
```
such as
```
              CALL_OPERATOR(
                  {nextOp->traceInput(intermediateResult); nextOp->addInput(intermediateResult)},
                  nextOp,
                  curOperatorId_ + 1,
                  kOpMethodAddInput);
```
to make it part of driver execution framework

Reply:
Keep the current implementation per our discussion.